src/commands/cmd_hash.cc

```diff
@@ -258,6 +259,36 @@ class CommandHMSet : public Commander {
   std::vector<FieldValue> field_values_;
 };
 
+class CommandHSetEx : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    if (args.size() < 5) {
+      return {Status::RedisParseErr, errWrongNumOfArguments};
+    }
+    ttl_ = GET_OR_RET(ParseInt<uint64_t>(args[2], 10));
+    for (size_t i = 3; i < args_.size(); i += 2) {
+      field_values_.emplace_back(args_[i], args_[i + 1]);
+    }
```

Comment:
We should considered invalid input like `HSETEX key ttl k1 v1 k2` and reject it.

Reply:
addressed via f9551a6071bd5612b7a4633ff82eaa78a8a21798